lib/cuda_color_spinor_field.cpp

```diff
@@ -686,23 +697,23 @@ namespace quda {
       if (src.FieldOrder() == QUDA_PADDED_SPACE_SPIN_COLOR_FIELD_ORDER) {
         // special case where we use mapped memory to read/write directly from application's array
         void *src_d;
-        cudaError_t error = cudaHostGetDevicePointer(&src_d, const_cast<void*>(src.V()), 0);
-        if (error != cudaSuccess) errorQuda("Failed to get device pointer for ColorSpinorField field");
+        qudaError_t error = qudaHostGetDevicePointer(&src_d, const_cast<void *>(src.V()), 0);
```

Comment:
Probably don't need these error checks here since the quda wrapper will have this built in I imagine?